service/telemetry.go

```diff
@@ -163,6 +163,8 @@ func (tel *telemetryInitializer) initMetrics(res *resource.Resource, logger *zap
 		opts = append(opts, sdkmetric.WithReader(r))
 	}
 
+	opts = append(opts, telemetry.ViewOptionsFromConfig(cfg.Metrics.Views)...)
```

Comment:
Does this mean that views are only coming from the user-provided config, and not from components? I understand this is not the final state, but could you add some documentation on how to use this?

Reply:
That's a good point. I'll have to spend a bit of time trying to figure out how to give components the ability to create their own views.